Wikipedia/Code/ExploreViewController.swift

```diff
@@ -1242,6 +1312,12 @@ extension ExploreViewController {
         if !UIAccessibility.isVoiceOverRunning {
             presentImageRecommendationsFeatureAnnouncementIfNeeded()
         }
+        
+        let imageRecommendationsDataController = WMFImageRecommendationsDataController()
+        
+        if imageRecommendationsDataController.hasPresentedFeatureAnnouncementModal {
+            presentImageRecommendationsAnnouncementAltText()
+        }
```

Comment:
@l-olson1214 Can you move this bit up into the !isVoiceOverRunning conditional?

Reply:
Got it!